src/App.js

```diff
@@ -1,11 +1,17 @@
 import React from "react";
 import Header from "./components/Header.js";
+import { Route } from "react-router-dom";
 
+import WelcomePage from "./components/WelcomePage";
```

Comment:
There's a small problem with the welcome page. You forgot to add a link to the characters route.